server/api/src/main/scala/com/prisma/api/database/DatabaseApiTestDatabaseMutationBuilder.scala

```diff
@@ -0,0 +1,123 @@
+package com.prisma.api.database
+
+import com.prisma.shared.models.TypeIdentifier.TypeIdentifier
+import com.prisma.shared.models.{Model, TypeIdentifier}
+import slick.dbio.DBIOAction
+import slick.jdbc.MySQLProfile.api._
+
+object DatabaseApiTestDatabaseMutationBuilder {
```

Comment:
If this is indeed just used in tests, it should reside in `src/test/scala`

Reply:
moved it